torch/_inductor/cpp_builder.py

```diff
@@ -1521,7 +1520,7 @@ def format_build_command(
     def get_target_file_path(self) -> str:
         return normalize_path_separator(self._target_file)
 
-    def build(self) -> Tuple[bytes, str]:
+    def build(self) -> None:
         """
         It is must need a temperary directory to store object files in Windows.
```

Comment:
@Skylion007: This is a little off-topic, but I see two issues in `CppBuilder.build()`.

The first is that the docstring is not really useful.

The second is that the temporary directory doesn't get actually get deleted if `run_compile_cmd` raises an exception. 

If this is intended behavior, it should be documented; otherwise, the `_remove_dir` should be in a `finally:` block.

I'd be happy to fix both of these while I was here.